modules/oe_webtools_social_share/oe_webtools_social_share.module

```diff
@@ -0,0 +1,15 @@
+<?php
+
+/**
+ * @file
+ * The OpenEuropa Webtools Social Share module.
+ */
+
+declare(strict_types = 1);
+
+/**
+ * Implements hook_locale_translation_projects_alter().
+ */
+function oe_webtools_social_share_locale_translation_projects_alter(&$projects) {
+  $projects['oe_webtools_social_share']['info']['interface translation server pattern'] = \Drupal::service('extension.list.module')->getPath('oe_webtools_social_share') . '/translations/%project-%language.po';
```

Comment:
I wonder if we shouldn't put the translations in a subfolder so if we add more translations in the future they are in separate folders and thus easier to manage (so it would be /translations/%project/%language.po). Let me know what you think.

Reply:
Approach is interesting. For now we have more important problems with translation.